backoff_tests.py

```diff
@@ -175,7 +298,7 @@ def test_on_exception_success():
                           on_success=log_success,
                           on_backoff=log_backoff,
                           on_giveup=log_giveup,
-                          jitter=lambda: 0,
+                          jitter=backoff.random_jitter,
```

Comment:
I don't think this needs to change... should still be jitter=None I think. The idea is to take the jitter out of the equation for this test.